og.routing.yml

```diff
@@ -43,3 +43,16 @@ og.remove_multiple_roles_confirm:
     _form: '\Drupal\og\Form\OgRemoveMultipleRolesForm'
   requirements:
     _custom_access: '\Drupal\og\Form\OgRemoveMultipleRolesForm::access'
+
+og.user_memberships:
+  path: '/user/{user}/groups'
+  defaults:
+    _entity_list: 'og_membership'
+    _title: 'My groups'
+  requirements:
+    user: '\d+'
+    _entity_access: 'user.view'
+  options:
+    parameters:
```

Comment:
```suggestion
    _admin_route: 'TRUE'
    parameters:
```

I suggest we add this as an admin route so it's themed using the admin theme.

<img width="678" alt="groups" src="https://user-images.githubusercontent.com/1823998/53885917-26c32d80-401f-11e9-8808-236b3a554c1d.png">
<img width="1388" alt="groups admin theme" src="https://user-images.githubusercontent.com/1823998/53885918-275bc400-401f-11e9-8c8b-b3212a6d76a4.png">